src/main/java/io/stargate/sgv2/jsonapi/config/constants/HttpConstants.java

```diff
@@ -0,0 +1,10 @@
+package io.stargate.sgv2.jsonapi.config.constants;
+
+public interface HttpConstants {
+
+  /** JSON API Authentication token header name. */
+  String AUTHENTICATION_TOKEN_HEADER_NAME = "Token";
+
+  /** JSON API also supports X-Cassandra-Token for backward compatibility. */
+  String DEPRECATED_AUTHENTICATION_TOKEN_HEADER_NAME = "X-Cassandra-Token";
```

Comment:
Do we have a test that tests the deprecated header still works? It looks like all the tests just use the `AUTHENTICATION_TOKEN_HEADER_NAME`